src/common/warning.rs

```diff
@@ -49,7 +52,7 @@ mod tests {
         );
 
         assert_eq!(
-            ".env:1 DuplicatedKey: The FOO key is duplicated",
+            ".env:\u{1b}[3m1\u{1b}[0m \u{1b}[1;31mDuplicatedKey\u{1b}[0m: The FOO key is duplicated",
```

Comment:
I would write something like this:
```suggestion
            format!(
                "{} {} {}",
                format!("{}:{}", ".env", "1").italic(),
                "DuplicatedKey:".red().bold(),
                "The FOO key is duplicated"
            ),
```